_site/standards/html-css/index.html

```diff
@@ -0,0 +1,193 @@
+
+        <header class="header header--full-width">
+  <div class="header__container">
+    <div class="header__brand">
+        <a href="/">
+        
+        <span class="header__title">
+          Justice On The Web
+          
+        </span>
+        </a>
+    </div>
+
+    
+      <div data-module="navigation">
+        <button type="button" class="header__navigation-toggle js-nav-toggle" aria-controls="navigation" aria-label="Show or hide top level navigation">Menu</button>
+
+        <nav id="navigation" class="header__navigation js-nav" aria-label="Top Level Navigation" aria-hidden="true">
+          <ul>
+            
+              <li>
+                <a href="https://github.com/ministryofjustice/justice-on-the-web">
+                  GitHub
+                </a>
+              </li>
+            
+          </ul>
+        </nav>
+      </div>
+    
+  </div>
+</header>
+
+      </div>
+
+      <div id="toc-heading" class="toc-show fixedsticky">
+        <a href="#toc" class="toc-show__label js-toc-show" aria-controls="toc">
+          Table of contents <span class="toc-show__icon"></span>
+        </a>
+      </div>
+
+      <div class="app-pane__body" data-module="in-page-navigation">
+        <div class="app-pane__toc">
+          <div class="toc" data-module="table-of-contents">
+            <a href="#" class="toc__close js-toc-close" aria-controls="toc" aria-label="Hide table of contents"></a>
+            <nav id="toc" class="js-toc-list toc__list" aria-labelledby="toc-heading">
+              
+
+<ul>
+  <li><a href="#html-and-css">HTML and CSS</a>
+    <ul>
+      <li><a href="#html5">HTML5</a></li>
+      <li><a href="#css">CSS</a>
+        <ul>
+          <li><a href="#flexbox">Flexbox</a></li>
+          <li><a href="#css-grid">CSS GRID</a></li>
+          <li><a href="#sass">SASS</a></li>
+          <li><a href="#bem">BEM</a></li>
+        </ul>
+      </li>
+      <li><a href="#image-use">Image use</a></li>
+      <li><a href="#webpack">Webpack</a>
+        <ul>
+          <li><a href="#laravel-mix">Laravel Mix</a></li>
+        </ul>
+      </li>
+    </ul>
+  </li>
+</ul>
+
+
+            </nav>
+          </div>
+        </div>
+
+        <div class="app-pane__content toc-open-disabled">
+          <main id="content" class="technical-documentation" data-module="anchored-headings">
+            <h1 id="html-and-css">HTML and CSS</h1>
+
+<h2 id="html5">HTML5</h2>
+
+<p>Our team uses HTML5 language features as default, when possible, building the websites html markup.</p>
+
+<p>Example using semantic elements:</p>
+<p align="left"><img src="https://www.w3schools.com/html/img_sem_elements.gif " /></p>
+
+<p>Useful resources</p>
+<ul>
+  <li><a href="https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/HTML5">Mozilla Offical HTML5 Docs</a></li>
+</ul>
+
+<h2 id="css">CSS</h2>
+
+<p>When possible, use flexbox and CSS grid to achieve frontend page layouts.</p>
```

Comment:
Flexbox can have some issues with layout in Internet Explorer and slightly older versions of browsers. Should our default be float-based layouts and then enhance with Flexbox/Grid?

Reply:
Good question. I think really need to crunch the numbers here on what browsers we support on what sites. I'd really like to avoid float-based layouts, especially going forward if possible. Agreed though this section needs to be pinned down with more data and thought.

Reply:
Maybe that research can be an action for our standards meetings :D it would be interesting to look into the analytics and see what's in use.